api/.gitignore

```diff
@@ -0,0 +1,3 @@
+.env.local
+**/dist/
+node_modules/
```

Comment:
@lolitaroz should I remove this and have you add it as a fix on main before I rebase? Including node_modules probably makes my pr harder to review

Reply:
yes that would be great, please don't commit node_modules lol!

Reply:
i added overall git ignore pushes so that should be available too, making yaml now